### ISSUE TITLE
[Bug]: Packer Unknown source type hcloud

### ISSUE THREAD

**Issue by b-reich** [kube-hetzner/terraform-hcloud-kube-hetzner#835]
### Description

running `tmp_script=$(mktemp) && curl -sSL -o "${tmp_script}" https://raw.githubusercontent.com/kube-hetzner/terraform-hcloud-kube-hetzner/master/scripts/create.sh && chmod +x "${tmp_script}" && "${tmp_script}" && rm "${tmp_script}"` failed.

```bash
Enter the name of the folder you want to create (leave empty to use the current directory instead, useful for upgrades):

The snapshots are required and deployed using packer. If you need specific extra packages, you need to choose no and edit hcloud-microos-snapshots.pkr.hcl file manually. This is not needed in 99% of cases, as we already include the most common packages.

Do you want to create the MicroOS snapshots (we create one for x86 and one for ARM architectures) with packer now? (yes/no): yes
Enter your HCLOUD_TOKEN: <mytoken>

Running: packer build packer build hcloud-microos-snapshots.pkr.hcl
Error: Unknown source type hcloud

  on hcloud-microos-snapshots.pkr.hcl line 99:
  (source code not available)

known builders: [docker amazon-ebs vagrant amazon-chroot qemu virtualbox-ovf
parallels-iso amazon-ebssurrogate vmware-iso amazon-ebsvolume parallels-pvm
vsphere-iso azure-dtl azure-arm amazon-instance vsphere-clone virtualbox-vm
googlecompute null virtualbox-iso azure-chroot file vmware-vmx]

Error: Unknown source type hcloud

  on hcloud-microos-snapshots.pkr.hcl line 128:
  (source code not available)

known builders: [virtualbox-vm googlecompute null virtualbox-iso amazon-instance
vsphere-clone file vmware-vmx azure-chroot vagrant amazon-chroot qemu
virtualbox-ovf parallels-iso amazon-ebssurrogate docker amazon-ebs parallels-pvm
vsphere-iso azure-dtl azure-arm vmware-iso amazon-ebsvolume]



Remember, don't skip the hcloud cli, to activate it run 'hcloud context create <project-name>'. It is ideal to quickly debug and allows targetted cleanup when needed!

Before running 'terraform apply', go through the kube.tf file and fill it with your desired values.
```

### Kube.tf file

```terraform
not a tf problem
```

### Platform

Mac

**Comment by b-reich** [comment]
<img width="772" alt="grafik" src="https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/assets/1573025/df321804-cbbf-48ef-9b46-7a4003afc99e">

https://github.com/hashicorp/packer/releases/tag/v1.9.0

The documentation needs to be updated.
I can provide a pr :)

**Comment by mysticaltech** [comment]
Thanks for finding that out @b-reich!

**Comment by b-reich** [comment]
> Thanks for finding that out @b-reich!

No Problem. This should be released soon :)

**Comment by mysticaltech** [comment]
Indeed! It's been shipped in v2.2.1 🙏
